packages/ui-kit/src/components/Logger/LogProvider.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { createContext } from 'react'
+
+import { LogLevel } from './Log.types'
+
+export type LogContextProps = {
+  logLevel: LogLevel
+}
+
+export const LogContext = createContext({ logLevel: LogLevel.Error })
```

Comment:
The LogContextProps looks unused. I think we should either (1) delete it, or (2) use it like this, no?

```suggestion
export type LogContextProps = {
  logLevel: LogLevel
}

export const LogContext = createContext<LogContextProps>({ logLevel: LogLevel.Error })
```

If we can delete it, that's even better, because we avoid introducing more types.

Reply:
Yea, somehow it was deletd. But adding it now.